vue/components/ui/atoms/breadcrumbs/breadcrumbs.vue

```diff
@@ -2,13 +2,26 @@
     <div class="breadcrumbs">
         <template v-for="(breadcrumb, index) in breadcrumbs">
             <template v-if="index !== breadcrumbs.length - 1">
+                <router-link
+                    v-bind:to="breadcrumb.href"
+                    v-if="typeof breadcrumb.href === 'object'"
+                    v-bind:key="index"
+                >
+                    <link-ripe
+                        class="breadcrumb-link"
+                        v-bind:text="breadcrumb.text"
+                        v-bind:hover="'color'"
+                        v-bind:style="partStyle"
+                    />
+                </router-link>

```

Comment:
Use something like this:

```
                    <router-link
                        v-bind:to="{
                            name: 'context-edit',
                            params: { name: item.name },
                            query: { ...$route.query }
                        }"
                        v-slot="{ href, navigate }"
                    >
                        <link-ripe v-bind:text="String(item.name)" v-bind:href="href" v-bind:hover="'border'" v-on:click="navigate" />
                    </router-link>
```

The point is binding the `link-ripe` `href` and `v-on:click` based on what comes from `router-link`.